cypress/e2e/tests/customMetrics/metrics_application_inventory.test.ts

```diff
@@ -0,0 +1,51 @@
+import {
+    createMultipleApplications,
+    deleteApplicationTableRows,
+    login,
+    preservecookies,
+} from "../../../utils/utils";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+
+let applicationList: Array<Assessment> = [];
+let metrics = new Metrics();
+
+describe(["@tier2"], "Metrics konveyor_applications_inventoried", function () {
+    before("Login and create test data", function () {
+        // Perform login
+        login();
+
+        // Save the session and token cookie for maintaining one login session
+        preservecookies();
+
+        // Navigate to Application inventory tab, delete all applications
+        deleteApplicationTableRows();
+
+        // Create 5 applications
+        applicationList = createMultipleApplications(5);
+    });
+
+    it("Validate metrics application inventoried count increased", function () {
+        let count = applicationList.length;
+
+        // Reload the metrics page
+        metrics.reloadPage();
+
+        // Validate the application inventory count increased
+        metrics.validateApplicationsInventoried(count);
+    });
+
```

Comment:
Third Test Case

`it("validate metrics does not change on discard assessment ", function () {`